content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -15,7 +15,7 @@ weight: 70
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
```

Comment:
This change has the meaning that the original documents were incorrect and dual-stack did not go alpha until v1.17

Is that correct?

Reply:
That is incorrect and wasn't my intent. This enhancement went into alpha in 1.16 and is still alpha in 1.18. I was trying to make that clear in the documentation as I have been asked a number of times if this is beta. If this isn't the change that needs to happen then I will remove it. Is there any way to say that it's still in alpha in 1.18?

Reply:
Please leave the original line as-is. Separately, it might be worth revising how the shortcode renders.